internal/pkg/ccloudv2/kafkarest.go

```diff
@@ -58,15 +58,17 @@ func (c *KafkaRestClient) UpdateKafkaClusterConfigs(clusterId string, req kafkar
 	return kafkarest.NewError(c.GetUrl(), err, httpResp)
 }
 
-func (c *KafkaRestClient) BatchCreateKafkaAcls(clusterId string, list kafkarestv3.CreateAclRequestDataList) (*http.Response, error) {
-	return c.ACLV3Api.BatchCreateKafkaAcls(c.context(), clusterId).CreateAclRequestDataList(list).Execute()
+func (c *KafkaRestClient) BatchCreateKafkaAcls(clusterId string, list kafkarestv3.CreateAclRequestDataList) error {
+	httpResp, err := c.ACLV3Api.BatchCreateKafkaAcls(c.context(), clusterId).CreateAclRequestDataList(list).Execute()
+	return kafkarest.NewError(c.GetUrl(), err, httpResp)
 }
 
-func (c *KafkaRestClient) CreateKafkaAcls(clusterId string, data kafkarestv3.CreateAclRequestData) (*http.Response, error) {
-	return c.ACLV3Api.CreateKafkaAcls(c.context(), clusterId).CreateAclRequestData(data).Execute()
+func (c *KafkaRestClient) CreateKafkaAcls(clusterId string, data kafkarestv3.CreateAclRequestData) error {
+	httpResp, err := c.ACLV3Api.CreateKafkaAcls(c.context(), clusterId).CreateAclRequestData(data).Execute()
+	return kafkarest.NewError(c.GetUrl(), err, httpResp)
```

Comment:
Thoughts on not including this in the error?
```suggestion
	return kafkarest.NewError(err, httpResp)
```

Reply:
I think it could provide useful information, so I'm not sure about removing it.